pytorch_lightning/loggers/base.py

```diff
@@ -168,6 +168,31 @@ def _convert_params(params: Union[Dict[str, Any], Namespace]) -> Dict[str, Any]:
 
         return params
 
+    @staticmethod
+    def _sanitize_callable_params(params: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Sanitize callable params dict, e.g. ``{'a': <function_**** at 0x*****> -> {'a': 'function_****'}``.
+
+        Args:
+            params: Dictionary containing the hyperparameters
+
+        Returns:
+            dict.
+        """
+        def _sanitize_callable(val):
```

Comment:
Is it necessary to give callables a chance to return? i.e why not just use `val.__name__` regardless?

Reply:
Just a design choice :) Possible a function doesn't take any arguments and will return a variable.

Reply:
I do not think the values is needed, it an even higher chance that it may crash for missing arguments...